src/libs/actions/BankAccounts.js

```diff
@@ -554,7 +555,19 @@ function validateBankAccount(bankAccountID, validateCode) {
                 API.User_IsUsingExpensifyCard()
                     .then(({isUsingExpensifyCard}) => {
                         Onyx.merge(ONYXKEYS.USER, {isUsingExpensifyCard});
-                        Onyx.merge(ONYXKEYS.REIMBURSEMENT_ACCOUNT, {loading: false, error: ''});
+                        Onyx.merge(ONYXKEYS.REIMBURSEMENT_ACCOUNT, {
```

Comment:
Instead of batching two `Onyx.merge()` could we just call this once like so...

```js
const reimbursementAccount = {loading: false, error: '', achDAta: {state: BankAccount.STATE.OPEN}};

if (isUsingExpensifyCard) {
    Navigation.dismissModal();
} else {
    reimbursementAccount.achData.currentStep = CONST.BANK_ACCOUNT.STEP.ENABLE;
}

Onyx.merge(ONYXKEYS.REIMBURSEMENT_ACCOUNT, data);
```

Reply:
Thanks for the suggestion, it does look a lot clearer!